src/purify.js

```diff
@@ -1183,13 +1182,14 @@ function createDOMPurify(window = getGlobal()) {
   };
 
   /**
-   * _basicCustomElementCheck
+   * _isBasicCustomElementCheck
    * checks if at least one dash is included in tagName, and it's not the first char
    * for more sophisticated checking see https://github.com/sindresorhus/validate-element-name
    *
    * @param {string} tagName name of the tag of the node to sanitize
+   * @returns {boolean} Returns true if the tag name meets the basic criteria for a custom element, otherwise false.
    */
-  const _basicCustomElementTest = function (tagName) {
+  const _isBasicCustomElementCheck = function (tagName) {
     return tagName.indexOf('-') > 0;
   };
```

Comment:
I added the prefix `is` because the function differs from the function name in the `jsdoc`, and because it returns a boolean.

jsdoc: _basicCustomElementCheck -> `This seems to fit the intent better.`
actual:  _basicCustomElementTest